ucare/config.go

```diff
@@ -13,14 +13,11 @@ const (
 	simpleAuthScheme    = "Uploadcare.Simple"
 	signBasedAuthScheme = "Uploadcare"
 	dateHeaderFormat    = time.RFC1123
+
+	signedUploadTTL = 60 * time.Second
```

Comment:
It should be configurable (probably in another PR)

Reply:
not configurable in pyuploadcare, but probably you are right
but yeah, separate PR, not now